diff -Nur wicd-1.5.1.orig/wicd/gui.py wicd-1.5.1/wicd/gui.py
--- wicd-1.5.1.orig/wicd/gui.py	2008-08-03 21:57:55.000000000 -0500
+++ wicd-1.5.1/wicd/gui.py	2008-08-04 11:42:34.429433719 -0500
@@ -842,7 +842,7 @@
         profile = self.combo_profile_names.get_active_text()
         try:
             sudo_prog = misc.choose_sudo_prog()
-            msg = "You must enter your password to configure scripts"
+            msg = "You must enter root's password to configure scripts"
             if sudo_prog.endswith("gksu") or sudo_prog.endswith("ktsuss"):
                 msg_flag = "--message"
             else:
@@ -1130,7 +1130,7 @@
         """ Launch the script editting dialog. """
         try:
             sudo_prog = misc.choose_sudo_prog()
-            msg = "You must enter your password to configure scripts"
+            msg = "You must enter root's password to configure scripts"
             if sudo_prog.endswith("gksu") or sudo_prog.endswith("ktsuss"):
                 msg_flag = "-m"
             else:
